conda-store-server/conda_store_server/plugins/plugin_manager.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) conda-store development team. All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+import pluggy
+
+from conda_store_server.exception import CondaStorePluginNotFoundError
+from conda_store_server.plugins import BUILTIN_PLUGINS, types
+
+
+class PluginManager(pluggy.PluginManager):
```

Comment:
Can we have a docstring for this subclass explaining the need for extra methods?